public/components/trace_analytics/components/common/plots/service_map_scale.tsx

```diff
@@ -21,9 +21,11 @@ export function ServiceMapScale(props: {
   const getScaleData = () => {
     const ticks = props.ticks;
     const delta = ticks[1] - ticks[0];
-    const title = { latency: 'Latency (ms)', error_rate: 'Error rate', throughput: 'Throughput' }[
-      props.idSelected
-    ];
+    const title = {
+      latency: 'Average duration (ms)',
+      error_rate: 'Error rate',
+      throughput: 'Request rate',
```

Comment:
Something feels slightly off with having units on only one of these -- the other rates aren't clearly (errors/s) or (requests/min) or something. I also would say "Latency" is a more accurate technical term than "Duration" but it's probably not a huge deal.

Reply:
Added the labels on the y axis lablels only - otherwise kept consistent